seed/tests/data/property_sample_data.json

```diff
@@ -225,4 +225,27 @@
   "owner_city_state": "Colorado",
   "owner_postal_code": 80202,
   "organization_id": -1
+}, {
+  "jurisdiction_property_id": 123481,
+  "pm_parent_property_id": "5766981",
+  "pm_property_id": "5766981",
+  "property_type": "Office",
```

Comment:
Any logic to assigning the IDs? the pm_property_id is incremented by 2 from the sample property above

Reply:
I think that's fine